osu.Game/Screens/Edit/Editor.cs

```diff
@@ -1256,6 +1256,15 @@ private IEnumerable<MenuItem> createFileMenuItems()
                 yield return externalEdit;
             }
 
+            if (editorBeatmap.BeatmapInfo.OnlineID > 0)
+            {
+                yield return new OsuMenuItemSpacer();
+                yield return new EditorMenuItem(EditorStrings.OpenInfoPageInBrowser, MenuItemType.Standard,
+                    () => (Game as OsuGame)?.OpenUrlExternally(editorBeatmap.BeatmapInfo.GetOnlineURL(api, editorBeatmap.BeatmapInfo.Ruleset)));
```

Comment:
Thoughts on having this open the overlay in game, since that already exists (and a second click on the external link helper icon can take you to the site)?

I'd probably also drop the "in browser" suffix from both text lines for brevity.

Reply:
> Thoughts on having this open the overlay in game, since that already exists

Not sure. The big problem I have with this proposal is that from the submission angle, the number 1 usage of this I can see is for editing the description or the other online metadata of the beatmap (like category), which is not possible from the overlay, so it'd just be an extra click. Not sure anyone needs to just see the overlay read-only from inside editor.

Reply:
Fair enough I guess.